features/tests/test_api.py

```diff
@@ -341,6 +343,35 @@ def test_feature_opening_hours(snapshot, api_client):
     snapshot.assert_match(executed)
 
 
+def test_price_list(snapshot, api_client):
+    PriceTagFactory(price=Decimal("100.01"), unit="a month")
+    PriceTagFactory(price=Decimal("200.01"), unit="a year")
+
+    executed = api_client.execute(
+        """
+    query FeaturePriceList {
+          features{
+            edges {
+              node {
+                type
+                properties {
+                  details{
+                    priceList{
+                      item
+                      price
+                      unit
+                    }
+                  }
+                  }
+                }
+              }
+            }
+          }
```

Comment:
indents seem to be off in a few places here

Reply:
and spaces missing after `details` and `priceList` and before the opening curly braces

Reply:
some spaces added. indents seem fine to black in the default mode.

Reply:
black does not check indents within multiline strings.

Reply:
to me indents look fine, could you show what you suggest changing?  
the best practices doc suggests that we enforce black, so I don't think that changing the formatting where both black and flake8 do not consider it necessary is needed.

Reply:
started a new thread. as I said, it is not related to python, black or flake8 in any way, since it is just within the string.

Reply:
true. I thought that black does at least some reformatting for a string that contains code.